general-terms-of-use.md

```diff
@@ -1,48 +1,48 @@
 
 We continually update our Services to offer the best possible product. While we plan to continue to offer and improve our Services, we may, without prior notice, change, cancel, create usage limits for, or permanently or temporarily stop offering or allowing you access to the Services generally (“Service Changes”). While we’ll try to give you advance notice of Service Changes that will adversely affect you, this might not be practical or possible and we retain the right to make Service Changes without notice and liability for any reason. If this Agreement ends you continue to be bound by it in any interactions you may have with the Services.
 
 You are fully responsible for your interactions with other Users, and we have no liability for your interactions with any other Users, or for any User’s actions or inactions with respect to you.
 
 ### SECTION 2. RIGHTS IN OUR CONTENT
 
-“Student Data” is any information (in any format) that is directly related to any identifiable current or former student that is maintained by a School, and may include “educational records” as defined by the Family Educational Rights and Privacy Act (“FERPA”). While we may need to access Student Data to provide the Services to you, between us, you own the Student Data and remain responsible for it. Our [Additional Terms of Use for Schools](https://clever.com/about/term) and [Privacy Policy](https://clever.com/about/privacy-policy) provide more detail around how we handle Student Data.
+“Student Data” is any information (in any format) that is directly related to any identifiable current or former student that is maintained by a School, and may include “educational records” as defined by the Family Educational Rights and Privacy Act (“FERPA”) that Schools provide to us. While we may need to access Student Data to provide the Services to you, Schools own the Student Data and remain responsible for it. Our [Additional Terms of Use for Schools](https://clever.com/about/term), [Additional Terms of Use for Developers](https://clever.com/about/terms-developers), and [Privacy Policy](https://clever.com/about/privacy-policy) provide more detail around how we handle Student Data.
```

Comment:
Link for ToU for Schools should point to /terms-schools. Right now it's linked to the general /terms page